### ISSUE TITLE
Why are modversions ordered by number and not by date

### ISSUE THREAD

**Issue by fcpwiz** [TechnicPack/TechnicSolder#538]
Since modversions are ordered by number the modversions are all out of order. For example, in a list ordered from oldest to newest modversions, 4.2.19 comes before 4.2.2 (because 1 is smaller than 2, but 19 isn't smaller than 2). It should be the other way around, that's why I think modversions should be ordered by date and not by number.

<img width="182" alt="screen shot 2016-01-09 at 5 50 25 pm" src="https://cloud.githubusercontent.com/assets/2921172/12218849/7ee250a2-b6f9-11e5-919a-5e36d02f7513.png">
Look, 1.2.12 is the latest version, though it's in the middle of the list. This makes it easy to make a dumb mistake when picking the correct modversion.

**Comment by spannerman79** [comment]
Dupe #492 - sort of

**Comment by fcpwiz** [comment]
@spannerman79 Yeah I have that issue too. Really really annoying. Seems like an easy fix if all builds were dated and sorted by that and not by number.

**Comment by spannerman79** [comment]
Only issue is the weird way that some mod devs number their build versions.  Not to mention sometimes a modder (or two) just re-uploads the same version number when clearly there has been changes made.  CustomNPC is one such mod :frowning:

**Comment by Isigiel** [comment]
This
https://github.com/TechnicPack/TechnicSolder/blob/master/app/views/modpack/build/view.blade.php#L85
should be changed to sort by updated_at.
